src/app/public/modules/page-summary/page-summary-alert.component.ts

```diff
@@ -1,5 +1,9 @@
 import { Component } from '@angular/core';
 
+/**
+ * Specifies messages that require immediate attention as alerts within
+ * the page summary.
```

Comment:
@Blackbaud-ToddRoberts, are you good with deleting the examples in the current docs? Makes sense to me, but just checking with you since I think you wrote the examples that we have in place now: `For example, you can display system-generated messages when certain criteria are met, or you can display notes about a record that you enter manually.`

Reply:
Yeah we can get rid of those two examples

Reply:
Actually, looking at the preview I think we should move each of these items to the design tab. Can get rid of the second example here, since we don't recommend manually-entered alerts even though that exists in a few products.